scripts/get_cert.sh

```diff
@@ -18,6 +18,7 @@ ADDRESS=$1
 SUBJECT=$2
 PROXY=$3
 
+sleep 10
```

Comment:
why we need sleep here? 
if it's for prepare_test_env, then we should add sleep just before running prepare_test_env
if it's for test_imm (fetchng server cert), then we should increase this sleep: https://github.com/IntelAI/inference-model-manager/blob/cb63d7dea53265b3984bf129529bad0bb38fb0d0/scripts/test_imm.sh#L105

Reply:
It is some kind of workaround as apparently certs needs some time to propagate and without this sleep, fetching this cert will fail. In previous code this script was called some time later (main.sh > prepare_test_env.sh > get_cert.sh), now it is called from main.sh directly, so we are missing some secs. I believe this sleep could be also 5 secs or smth. I will move it before running prepare_test_env